src/screens/ApprovedSubjects/index.js

```diff
@@ -0,0 +1,33 @@
+
+import React, { Component } from 'react';
+import { AppRegistry, FlatList, StyleSheet, Text, View } from 'react-native';
+
+export default class ApprovedSubjects extends Component {
```

Comment:
you can export this class as a `export default ApprovedSubjects;` at the bottom of the file